tasks/utilities/task_validate.wdl

```diff
@@ -54,256 +54,56 @@ task export_two_tsvs {
   }
 }
 
-task compare_two_tsvs {
+task theiavalidate {
   input {
-    String datatable1
     File datatable1_tsv
-    String datatable2
     File datatable2_tsv
-    File? validation_criteria_tsv
     String columns_to_compare
     String output_prefix
 
+    File? validation_criteria_tsv
+    File? column_translation_tsv
+    String? na_values
```

Comment:
I may have missed this in a previous sync, but do you mind explaining the utility of forcing values to NA?

Reply:
We ran into this issue when we were making a release and "NA" values were being converted to None when we actually wanted them to be "NA" and that caused a lot of problems. I think we came up with this solution together.